roles/compliance/tasks/run_compute_node_scans.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Install needed packages
+- name: Install openscap-scanner and security guide
+  become: true
+  ansible.builtin.package:
+    name:
+      - openscap-scanner
+      - scap-security-guide
+    state: present
+
+- name: Create folder to hold scan results
+  ansible.builtin.file:
+    path: "{{ cifmw_compliance_compute_artifacts_basedir }}"
+    state: directory
+    recurse: true
+    owner: "{{ ansible_user | default(lookup('env', 'USER')) }}"
+    group: "{{ ansible_user | default(lookup('env', 'USER')) }}"
+
+- name: Do compliance scans
+  ansible.builtin.command: >
+    oscap xccdf eval --report {{ cifmw_compliance_compute_artifacts_basedir }}/{{ profile }}.html \
+    --profile {{ profile}}  /usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml
+  become: true
+  failed_when: false
+  loop: "{{ cifmw_compliance_compute_profiles }}"
+  loop_control:
+    loop_var: profile
+  when: not cifmw_compliance_dry_run | bool
+
+- name: Set ownership on report files
+  ansible.builtin.file:
+    path: "{{ cifmw_compliance_compute_artifacts_basedir }}"
+    owner: "{{ ansible_user | default(lookup('env', 'USER')) }}"
+    group: "{{ ansible_user | default(lookup('env', 'USER')) }}"
```

Comment:
please add a `mode`

Reply:
mode doesn't make sense here.  This task is simply using file to recurse through the directory and change ownership of any files that are "root" to the ansible_user.   We're not creating any new files.